packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -419,13 +432,15 @@ const DialogueRecommendationRow = ({ targetUser, checkId, userIsChecked, userIsM
 
 const DialogueRecommendationRowComponent = registerComponent('DialogueRecommendationRow', DialogueRecommendationRow, {styles});
 const TopicSuggestionComponent = registerComponent('TopicSuggestion', TopicSuggestion, {styles});
+const CommentViewComponent = registerComponent('CommentView', CommentView, {styles});
```

Comment:
If you're now registering the `CommentView` component and wrapping it in the `styles` HoC, but it previously took `classes` as a prop, that means wherever you were using it before, you were passing in `classes` to it manually.  I don't see any diffs that reflect that having been changed/fixed so I'm a bit suspicious that something is broken here

Reply:
It's only used in two places (frontpage recommendations, and dialogue match form). I checked behavior in both before making the PR, and it seemed fine. 

Nonetheless, I fixed the issue (properly imported CommentView and removed classes argument), and thanks for catching!